server/server.go

```diff
@@ -1378,3 +1376,15 @@ func (s *Server) SaveTTLConfig(data map[string]interface{}, ttl time.Duration) e
 	}
 	return nil
 }
+
+func stringsEqual(a, b []string) bool {
```

Comment:
it will be better  if move the function to typeutil.

Reply:
ok.